package.json

```diff
@@ -11,6 +14,8 @@
   },
   "homepage": "https://github.com/18F/identity-dev-docs#readme",
   "dependencies": {
-    "identity-style-guide": "^1.1.0"
+    "identity-style-guide": "^1.1.0",
+    "uswds": "^2.0.2",
```

Comment:
Do we use uswds directly? I thought that came with as part of identity-style-guide?

Reply:
Yeah, I'm not sure! Does @nickttng know the answer?

AFAIK adding `uswds` here just puts it in node_modules.  The new npm task/script I added copies the `uswds` source into the same place that the README specifies.

🤔 @nickttng Should I be copying files from `node_modules/identity_style_guide`?

Reply:
So for my PR yesterday to update `identity-style-guide`, I only had to change that one node dependency to get the extra styles: https://github.com/18F/identity-dev-docs/pull/102

I'm gonna check out your branch and see how things build if I take out that dep

Reply:
https://github.com/18F/identity-dev-docs/pull/106

Reply:
That PR totally works for me.  I'm don't 100% get _how_ yet, but it's def simpler! Thanks Zach!

Reply:
@ctro We should be using `"identity-style-guide": "^2.1.6"` which currently uses `uswds@2.0.3`, so there's no need to list USWDS directly in the dependencies. Hope this clears up.

Reply:
Yes, this helps!  But I'm still a little 🤔 about how identity-style-guide and USWDS work together.  I'm going to put some time on our calendars so you can teach me about it ❤️